Conan.VisualStudio/Conan.VisualStudio.csproj

```diff
@@ -177,22 +182,23 @@
       <HintPath>..\packages\Microsoft.VisualStudio.Validation.15.3.32\lib\net45\Microsoft.VisualStudio.Validation.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.VC.Interfaces, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <Reference Include="Microsoft.VisualStudio.VC.Interfaces, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.VCCodeModel, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <Reference Include="Microsoft.VisualStudio.VCCodeModel, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.VCProject, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <Reference Include="Microsoft.VisualStudio.VCProject, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.VCProjectEngine, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <Reference Include="Microsoft.VisualStudio.VCProjectEngine, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
```

Comment:
If you decided that the plugin should be ported to VS2019 (16.0 is 2019, right?), then probably NuGet package references should be updated, too.

Reply:
Yes 16 == 2019. 

Extension still targets and supports 15 == 2017. Updating everything to 16 would mean it wouldn't work on 2017 anymore.

These specific references are not to NuGet packages but to libraries part of the installed Visual Studio workloads. By updating this references I can build the extension using VS2019 for which I installed the C++ workload, where my VS2017 does not have that workload installed and thus no version 15 of the library.

Probably best would be to revert to 15 since that is still the current version and I keep this switch to 16 locally for myself.

Reply:
@ForNeVeR i thought you moved to mars. w/b.
@sboulema can you share paste example of current output window content with new changes?
VSIX.logger looks like a nice feature.